### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jhenApit**
Merge Latest Updates

### CODE REVIEW

File: Basecode.WebApp/Views/Login/ResetPassword.cshtml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    <div class="container-fluid text-black w-75 mt-5 pt-5">
+                        <div class="top text-white text-center">
+                            <h2>Reset Password</h2>
+                        </div>
+                        <div class="form-floating mb-3">
+                            <input asp-for="Password" type="password" class="form-control" id="floatingInput"
+                                   placeholder="password">
+                            <label asp-for="Password" for="floatingInput" class="form-label">Password</label>
+                        </div>
+                        @*<div class="form-floating mb-3">
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ResetPassword.cshtml:28]: Please remove code smell

File: Basecode.WebApp/Views/Login/ResetPassword.cshtml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+}
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-8 bg-dark" id="login_body">
+            </div>
+            <div class="col-4 bg-black vh-100 pt-5">
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ResetPassword.cshtml:17]: remove blank space

File: Basecode.WebApp/Views/Login/ResetPassword.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+﻿@model Basecode.WebApp.Models.LoginModel
+
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ResetPassword.cshtml:3]: remove blank spaces

File: Basecode.WebApp/Views/Login/Index.cshtml [hunk 4]
```diff
@@ -1,4 +1,6 @@
-﻿@model Basecode.WebApp.Models.LoginModel
+@using Basecode.Data.Dtos
+@model HREmployeeUpdationDto
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/Index.cshtml:5]: remove blank spaces

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                        <div class="d-grid gap-2">
+                            <a class="text-center text-white mt-3" asp-controller="Login" asp-action="Index">Back to login</a>
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ForgotPassword.cshtml:50]: remove blank spaces

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+                    </div>
+                    <div class="container-fluid text-black w-75 mt-5 pt-5">
+                        <div class="top text-white text-center">
+                            <h2>Forgot Password</h2>
+                            <p>Enter your email to receive a password reset link</p>
+                        </div>
+                        <div class="form-floating mb-3">
+                            <input asp-for="Email" type="email" class="form-control" id="floatingInput"
+                                   placeholder="name@example.com">
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ForgotPassword.cshtml:34]: walay placeholder mo gawas ari
image

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-8 bg-dark" id="login_body">
+            </div>
+            <div class="col-4 bg-black vh-100 pt-5">
+
+                <form asp-action="ForgotPassword" class="needs-validation">
+                    <div class="d-md-flex justify-content-sm-end">
+                        <button type="button" class="btn-close btn-close-white" aria-label="Close" onclick="cancelForm()"></button>
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ForgotPassword.cshtml:25]: remove blank space

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        window.location.href = "/";
+    }
+</script>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-8 bg-dark" id="login_body">
+            </div>
+            <div class="col-4 bg-black vh-100 pt-5">
+
```
Comment (FaerieStar) [inline Basecode.WebApp/Views/Login/ForgotPassword.cshtml:21]: remove blank space

File: Basecode.WebApp/Controllers/LoginController.cs [hunk 9]
```diff
@@ -1,19 +1,43 @@
...
+        {
+            _service = service;
+        }
         public IActionResult Index()
         {
             return View();
         }
 
-        public async Task<IActionResult> Login([Bind("Email, Password")] LoginModel loginModel)
+        //public async Task<IActionResult> Login([Bind("Email, Password")] LoginModel loginModel)
```
Comment (FaerieStar) [inline Basecode.WebApp/Controllers/LoginController.cs:21]: remove code smell

**Review by FaerieStar — CHANGES_REQUESTED** [review]
All functions seems to be working, just remove some code smells and fix the placeholder email for forgot password.

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml (hunk 6 above)
Comment (takennnusernames) [reply Basecode.WebApp/Views/Login/ForgotPassword.cshtml:34]: script ni sa bootstrap. hidden ang placeholder pero needed para sa floating
image
image

Comment (FaerieStar) [reply Basecode.WebApp/Views/Login/ForgotPassword.cshtml:34]: Sige sige noted

File: Basecode.WebApp/Views/Login/ResetPassword.cshtml (hunk 2 above)
Comment (takennnusernames) [reply Basecode.WebApp/Views/Login/ResetPassword.cshtml]: resolved in hotfix branch

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml (hunk 7 above)
Comment (takennnusernames) [reply Basecode.WebApp/Views/Login/ForgotPassword.cshtml]: resolved in hotfix branch

File: Basecode.WebApp/Views/Login/ForgotPassword.cshtml (hunk 8 above)
Comment (takennnusernames) [reply Basecode.WebApp/Views/Login/ForgotPassword.cshtml]: resolved in hotfix branch

**Review by FaerieStar — APPROVED** [review]
lgtm!

**Pull request merged**